jaffa-core/source/java/org/jaffa/config/ApplicationResourceLoader.java

```diff
@@ -69,8 +69,12 @@
 	private static final Logger log = Logger.getLogger(ApplicationResourceLoader.class);
 
 	public static final String PROP_APPLICATION_RESOURCES_DEFAULT = "ApplicationResourcesDefault";
+	public static final String PROP_APPLICATION_RESOURCES_DEFAULT_OVERRIDE = "ApplicationResourcesDefaultOveridde";
```

Comment:
ApplicationResourcesDefaultOveridde should be ApplicationResourcesDefaultOverride